src/azure/main.tf

```diff
@@ -1,3 +1,24 @@
+locals {
+  ghes_name       = var.GHES_NAME
+  ghes_hostname   = var.GHES_HOSTNAME
+  oidc_issuer_uri = "https://${local.ghes_hostname}/_services/token"
+
+  # Azure
+  azure_subscription_id                  = var.AZURE_SUBSCRIPTION_ID
+  azure_region                           = var.AZURE_REGION
+  azure_storage_account_tier             = var.AZURE_STORAGE_ACCOUNT_TIER
+  azure_storage_account_replication_type = var.AZURE_STORAGE_ACCOUNT_REPLICATION_TYPE
+  azure_blob_endpoint_suffix             = "core.windows.net"
+}
+
+resource "random_string" "long" {
```

Comment:
Since we use this across modules, we could have modules take in a random prefix as a variable, and generate the random prefix in our main.tf